apps/ui/components/requests/List.vue

```diff
@@ -10,15 +10,17 @@ const props = defineProps<{
 }>()
 
 const visibleRequests = ref(props.requests || [])
```

Comment:
Consider ensuring `props.requests` is reactive or watched to update `visibleRequests` accordingly. This can be achieved by using a `watch` or `watchEffect` on `props.requests` to update `visibleRequests` whenever the prop changes. This ensures that `visibleRequests` and consequently `hasOpenRequests` are always up to date. [important]